app/src/main/java/com/example/movieapp/crawler/pojo/Movie.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.movieapp.crawler.pojo
+
+import java.util.*
+
+/**
+ * Created by Benjamin Vouillon on 08,July,2020
+ */
+data class Movie(
+    val popularity: Float,
+    val vote_count: Int,
```

Comment:
snake_case isn't great for variables. Better use CamelCase. There's a way to tell Gson that voteCount refers to vote_count in the JSON file ;-)

Reply:
done